edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -145,6 +139,42 @@ def _update_container_config(self, path, config):
         os.chmod(path, 0o600)
         self.results['changed'] = True
 
+    def _tar_cleanup(self, tarinfo):
+        tarinfo.mtime = 0
+        return tarinfo
+
+    def _create_checksum(self, config_volume):
+        """Create an hash from a tar-ed version of the given
+        folder
+
+        :param config_volume: string
+        :returns: string
+        """
+
+        total_hash = hashlib.new('md5')
+        tmp = tempfile.TemporaryFile()
+        tar = tarfile.open(fileobj=tmp, mode="w:")
+        tar.add(config_volume, recursive=True, filter=self._tar_cleanup)
+        tar.close
+        tmp.flush()
+        tmp.seek(0)
+        tar = tarfile.open(fileobj=tmp, mode='r:')
+        chunk_size = 100 * 1024
```

Comment:
I'd define this as global constant instead of variable within the method.

Reply:
i changed completely the approach, see new commits :)